### DISCUSSION TITLE
Daily 27/04

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#173] [Daily Sprint]
- Qué hice ayer?
- Qué haré hoy?
- Dificultades o impedimentos

**Comment by NataliaAlvarezIspc** [comment]
Ayer comencè a trabajar en Angular con la parte de login y de registración. Subí al repo lo de registración. Además me estuve asignando algunas tareas más. 
Hoy contiruaré con el login y con la barra de navegación. Trataré de dejar esa parte lista para subir al repo. Por la tarde tengo clase sincrónica de Progaramación, así que espero llegar a hacer todo.

**Comment by BraianTroncoso** [comment]
Ayer tuve clases de Java, y apenas miré un poco de documentación. Hoy pienso pushear algo de lo que realicé por mi parte en Angular.

**Comment by rpgrca** [comment]
Ayer armé un componente para el alta de productos, probablemente hoy haga un para la modificación de productos pero no para el de bajas que debería ser un tachito de basura y una ventana de confirmación. Todo depende de cómo sea el catálogo ya que me imagino que el catálogo que vean los clientes sea parecido al catálogo nuestro.

**Comment by Brian-Almada** [comment]
Ayer estuve viendo un poco más de Angular. Hoy al fin pude crear el esqueleto (me animé a hacerlo). Ahora solo me falta rellenarlo con código para armar el catálogo.
